client/src/pages/locations/GeoLocation.js

```diff
@@ -7,15 +7,12 @@ import {
 } from "@blueprintjs/core"
 import * as FieldHelper from "components/FieldHelper"
 import { Field } from "formik"
-import {
-  convertLatLngToMGRS,
-  convertMGRSToLatLng,
-  parseCoordinate
-} from "geoUtils"
+import { convertLatLngToMGRS, convertMGRSToLatLng } from "geoUtils"
 import PropTypes from "prop-types"
 import React from "react"
 import { Col, ControlLabel, FormGroup, Table } from "react-bootstrap"
 import Settings from "settings"
+import { parseCoordinate } from "../../geoUtils"
```

Comment:
Why do you do the import like this for parseCoordinate? I this the import should have stayed as it was.

Reply:
It was probably auto import from ide when i was changing some stuff.